### DISCUSSION TITLE
Sound but no vision on downloads

### DISCUSSION THREAD

**Discussion by michaelahanti** [aclap-dev/video-downloadhelper#126] [Help and Q&A]
### Your question

I just bought a licence and so far I'm not impressed. I downloaded some videos from Vimeo. One worked fine, the rest had audio but never got beyond the first frame of the video. Perhaps I'm doing something wrong. Any advice would be appreciated

**Comment by jcfrog** [comment]
hi,
please give your versions and a link as an example. https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#start-a-new-qa-post

**Reply by michaelahanti** [reply]
Video DownloadHelper 8.1.4.0    https://vimeo.com/912060496/6d5b724312

**Reply by michaelahanti** [reply]
I'm using firefox on windows 11

**Reply by jcfrog** [reply]
Vimeo issue has been fixed with 8.2, you should update your extension.

**Reply by michaelahanti** [reply]
Thanks. I don't see an update button anywhere

**Reply by michaelahanti** [reply]
Is it safe, licence wise, to remove and reinstall?

**Reply by jcfrog** [reply]
Yes you can do that. You will just need to revalidate your license key.
If any problem we are here.

**Reply by michaelahanti** [reply]
no worries. I turned on automatic updates. I've been using this for a long time, and never had an issue. Only concerned it wasn't working with the licence. Thanks for your help
